pkg/container/resolver_test.go

```diff
@@ -104,8 +111,85 @@ func TestResolverFail(t *testing.T) {
 		common.ToPtr(""),
 		common.ToPtr(false),
 		false,
+		nil,
 	})
 	specs, err = resolver.Finish()
 	assert.Error(t, err)
 	assert.Len(t, specs, 0)
 }
+
+func TestResolverLocalManifest(t *testing.T) {
+	currentUser, err := user.Current()
+	assert.NoError(t, err)
+
+	if currentUser.Uid != "0" {
+		t.Skip("User is not root, skipping test")
+	}
+
+	_, err = exec.LookPath("podman")
+	if err != nil {
+		t.Skip("Podman not available, skipping test")
```

Comment:
For the dnf tests, we have a `--force-dnf` flag to make sure we don't skip the tests when we're certain they should be running.
https://github.com/osbuild/images/blob/a233d256318f70e86f891adb7abd4cf9da90f58f/pkg/dnfjson/dnfjson_test.go#L19-L27

This makes sure we don't skip the tests in CI if somehow something changes and the detection fails.
https://github.com/osbuild/images/blob/a233d256318f70e86f891adb7abd4cf9da90f58f/.github/workflows/tests.yml#L43-L44

Let's add a similar flag here.